core/systems/project/_assembly.ex

```diff
@@ -41,13 +43,41 @@ defmodule Systems.Project.Assembly do
     |> Repo.transaction()
   end
 
+  @spec create(any(), any(), :empty) :: any()
   def create(name, user, :empty) do
```

Comment:
Did you consider the use case on production where projects don't have storages yet?

Reply:
I did now! I created a second, separate, data tab which informs the user no storage exists for their project. This page also offers a button to create a new storage for their project. (it creates a storage item and a storage endpoint)